core/domain/collection_services.py

```diff
@@ -236,29 +299,28 @@ def get_multiple_collections_by_id(collection_ids, strict=True):
 
 
 def get_new_collection_id():
-    """Returns a new collection id."""
-    return collection_models.CollectionModel.get_new_id('')
-
+    """Returns a new collection id.
 
-def is_collection_summary_editable(collection_summary, user_id=None):
-    """Checks if a given user may edit an collection by checking
-    the given domain object.
+    Returns:
+        str. A new collection id.
     """
-    return user_id is not None and (
-        user_id in collection_summary.editor_ids
-        or user_id in collection_summary.owner_ids
-        or collection_summary.community_owned)
+    return collection_models.CollectionModel.get_new_id('')
 
 
 # Query methods.
 def get_collection_titles_and_categories(collection_ids):
     """Returns collection titles and categories for the given ids.
 
-    The result is a dict with collection ids as keys. The corresponding values
-    are dicts with the keys 'title' and 'category'.
+    Args:
+        collection_ids: list(str). Collection ids of the collection titles and
+        categories to be retrieved.
```

Comment:
indent by 4

Also, minor suggestion: "IDs of the collections whose titles and categories are to be retrieved." -- since it's the collections, and not the titles and categories, which have ids.